apps/interactor/tests/conftest.py

```diff
@@ -332,6 +336,7 @@ async def finish(self, exc_typ=None, exc=None, tb=None):
         color=hp.Color(100, 0, 1, 2500),
         group={"label": group_three_label, "identity": group_three_uuid},
         location={"label": location_two_label, "identity": location_two_uuid},
+        product=Products[VendorRegistry.LIFX, 22],
```

Comment:
this shouldn't be necessary

Reply:
Confirmed. Will remove the line from each block.